.github/workflows/check-links.yml

```
@@ -61,6 +62,8 @@ jobs:
           --exclude "^https://www.adobe.com/jp/print/postscript/pdfs/PLRM.pdf"
           --exclude "^https://www.generic-mapping-tools.org/remote-datasets/%s$"
           --exclude "^https://www.pygmt.org/%7B%7Bpath%7D%7D"
+          --exclude "^https://github.com/GenericMappingTools/pygmt/releases/download/%7B%7Bpath%7D%7D/pygmt-docs.pdf"
+          --exclude "^https://github.com/GenericMappingTools/pygmt/releases/download/%7B%7Bpath%7D%7D/pygmt-docs.zip"
           --exclude "^https://www.researchgate.net/"
           --exclude "^https://zenodo.org/badge/DOI/"
```

Do not check `.rst` and `.md` files under the `doc` directory, because these files will be built into HTML files and are already checked.
```suggestion
          --exclude "^https://zenodo.org/badge/DOI/"
          --exclude-path "repository/doc/**/*.rst"
          --exclude-path "repository/doc/**/*.md"
```